src/transformers/models/gemma3n/configuration_gemma3n.py

```diff
@@ -68,7 +68,7 @@ class Gemma3nTextConfig(PretrainedConfig):
             `num_key_value_heads=1` the model will use Multi Query Attention (MQA) otherwise GQA is used. When
             converting a multi-head checkpoint to a GQA checkpoint, each group key and value head should be constructed
             by meanpooling all the original heads within that group. For more details checkout this
-            [paper](https://arxiv.org/pdf/2305.13245.pdf). If not specified, will default to `num_attention_heads`.
+            [paper](https://huggingface.co/papers/2305.13245). If not specified, will default to `num_attention_heads`.
```

Comment:
Maybe this will fix the failing check?

```suggestion
            [paper](https://huggingface.co/papers/2305.13245). If not specified, will default to
            `num_attention_heads`.
```

Reply:
thanks!